src/covid_model_seiir_pipeline/pipeline/forecasting/model/containers.py

```diff
@@ -17,6 +17,63 @@
 )
 
 
+class Indices:
+    """Abstraction for building square datasets."""
```

Comment:
I'm familiar with the colloquial use of "square" but I'm confused as to what that means in Pandas given that all `DataFrame`s are square by definition (or maybe I'm mistaken and they aren't? they always provide a square `.shape`...)